lib/ts/types.ts

```diff
@@ -119,6 +122,16 @@ export type AppInfoUserInput = {
     apiGatewayPath?: string;
 };
 
+export type RecipeInitResult<T, C, R, P extends NormalisedRecipeModuleConfig<T, C, R>, I> = {
```

Comment:
Is there a case where the web-js Action type param does not match the auth-react Action type param? Is there a way to re-use it? It's fine if it doesn't work, I just wanted to check.

Reply:
they are the same type, which Action type should we use auth-react version or web-js ?